compiler/modules/nback/init.bal

```diff
@@ -51,6 +51,8 @@ type InitModuleContext record {|
     table<SubtypeDefn> key(typeCode, semType) subtypeDefns = table [];
     InitTypes llTypes;
     map<llvm:FunctionDecl> typeTestFuncs = {};
+    llvm:FunctionDecl? listSetFunc = ();
+    llvm:FunctionDecl? listGetFunc = ();
```

Comment:
Should I generalize these three into a single `map<llvm:FunctionDecl> runtimeFuncs = {};` ?

Reply:
No.